File: 221102_배열/11328_Strfry/11328.py
import sys

input = sys.stdin.readline
n = int(input())
for i in range(n):
    a, b = input().split()

    a = ''.join(sorted(a))
    b = ''.join(sorted(b))

    if len(a) != len(b):
        print('Impossible')
        continue

    for j in range(len(a)):
        if a[j] != b[j]:
            print('Impossible')
            break
    else:
        print('Possible')

File: 221102_배열/11328_Strfry/11328.java
import java.io.*;
import java.util.Arrays;


public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));


        int n = Integer.parseInt(br.readLine());
        for (int i = 0; i < n; i++) {
            String[] input = br.readLine().split(" ");

            char[] chars1 = input[0].toCharArray();
            char[] chars2 = input[1].toCharArray();
            Arrays.sort(chars1);
            Arrays.sort(chars2);

            String ans = "Impossible";
            if (Arrays.equals(chars1, chars2)) ans = "Possible";
            bw.write(ans + "\n");
        }
        bw.flush();
        bw.close();
    }
}